fhir-model/src/main/java/com/ibm/fhir/model/path/function/SubstringFunction.java

```diff
@@ -45,7 +45,7 @@ public int getMaxArity() {
         
         int start = getInteger(arguments.get(0));
         
-        if (start > value.length() - 1) {
+        if (start < 0 || start > value.length() - 1) {
```

Comment:
what's the behavior if you have a negative?

Reply:
Per the spec, we should return empty set if the start index is out of range.